Packs/CommonPlaybooks/Playbooks/playbook-Email_Address_Enrichment_-_Generic_v2.1.yml

```diff
@@ -397,8 +397,13 @@ tasks:
           - operator: uniq
       email:
         complex:
-          root: Account
-          accessor: Email
+          root: Account.Email
+          filters:
+          - - operator: isNotEmpty
```

Comment:
should we have 2 dashes here ?

Reply:
Yes see other occurrences in the file. For [example](https://github.com/demisto/content/blob/8cc72055c3ce5daab06074d10691e30a16c4946f/Packs/CommonPlaybooks/Playbooks/playbook-Email_Address_Enrichment_-_Generic_v2.1.yml#L107)